jmh/src/jmh/java/org/roaringbitmap/CheckedAddBenchmark.java

```diff
@@ -0,0 +1,89 @@
+package org.roaringbitmap;
+
+import org.openjdk.jmh.annotations.*;
+
+import java.util.concurrent.TimeUnit;
+
+@BenchmarkMode(Mode.AverageTime)
+@Warmup(iterations = 3, timeUnit = TimeUnit.MILLISECONDS, time = 2000)
+@Measurement(iterations = 5, timeUnit = TimeUnit.MILLISECONDS, time = 2000)
+@OutputTimeUnit(TimeUnit.MILLISECONDS)
+@State(Scope.Benchmark)
+public class CheckedAddBenchmark {
+
+  private static final int valuesCount = 10_000_000;
+  @Param({"HALF_DUPLICATED", "UNIQUE"})
+  public String inputData;
+
+  @Param({"RUN", "BITMAP"})
+  public String containerType;
+
+  public RoaringBitmap bitmap;
+
+  public int[] a;
+
+  @Setup(Level.Invocation)
+  public void setUp() {
+    a = new int[valuesCount];
+    if (inputData.equals("HALF_DUPLICATED")) {
+      for (int i = 0; i < valuesCount; i++) {
+        a[i] = i >> 2;
+      }
+    }
+    if (inputData.equals("UNIQUE")) {
+      for (int i = 0; i < valuesCount; i++) {
+        a[i] = i;
+      }
+    }
```

Comment:
just a style note here, it's nice to define an enum with values `UNIQUE`, `HALF_DUPLICATED` with routines that produce the data, it keeps the definitions of the scenarios, the implementations of the scenarios, and the rest of the setup code separated (imagine your eyes glazing over the the branches here if there were more than two scenarios).

Reply:
I like enums very much, but I want to create benchmark simple as possible. Ok, I will introduce enum for this purpose.

Reply:
Done in last commit.